### DISCUSSION TITLE
Masalah dalama docker-compose up

### DISCUSSION THREAD

**Discussion by wisnu45** [madeindra/codeigniter-microservice#8] [Q&A]
![image](https://user-images.githubusercontent.com/45603735/121967833-45d2cd00-cd9b-11eb-8900-21bcc451de9d.png)

Asslamulaikum Mas Made, mau nanya, sebelumnya pada 3 file tersebut pada /application/vendor Folder vendornya infalid ya..? apa bagaimana kok pas aku compose lagi berubah Mas Made..?

**Comment by madeindra** [comment]
@wisnu45 Sudah install dependency?
